src/coord/dist_aggregate.c

```diff
@@ -44,7 +44,7 @@ static bool getCursorCommand(long long cursorId, MRCommand *cmd, MRIteratorCtx *
   // command instead of a READ command (here we know it has more results)
   if (timedout && !cmd->forCursor) {
     newCmd = MR_NewCommand(4, "_FT.CURSOR", "DEL", idx, buf);
-    newCmd.depleted = true;
+    // newCmd.depleted = true;
```

Comment:
why commenting out?

Reply:
This is wrong. Here the cmd is essentially not depleted as we are going to send the shard another request...